src/components/nowplayingbar/nowplayingbar.js

```diff
@@ -405,10 +419,6 @@ define(['require', 'datetime', 'itemHelper', 'events', 'browser', 'imageLoader',
             muteButton.querySelector('i').innerHTML = 'volume_up';
         }
 
-        if (progressElement) {
-            progressElement.style.width = (volumeLevel || 0) + '%';
-        }
-
```

Comment:
why this is no longer needed?

Reply:
Slider updates itself: I call `updateValues` in `input` event handler (if `data-slider-keep-progress` is not `true`).